tests/test_import_export.py

```diff
@@ -83,9 +83,7 @@ def reset_rses():
 
 def test_active():
```

Comment:
My recommendation: remove the test altogether. It does not do anything useful. On the contrary, it tries to do things the testing framework was designed for i.e. selectively run/skip tests based on the environment.

Looking back at the commit where this  test was added, it looks more like a debug util for the developer working on the issue, that slipped through the cracks.
 
https://github.com/rucio/rucio/commit/e6bc1b3d6ba3c5f53ba85d425b12cf5ef4b28c1

Reply:
thanks for checking :) I'll go ahead and take it out